hepcrawl/spiders/elsevier_spider.py

```diff
@@ -46,24 +46,31 @@ class ElsevierSpider(XMLFeedSpider):
        it will yield a request to unzip them. Then for every record in
        the zip files it will yield a request to scrape them. You can also run
        this spider on a zip file or a single record file.
-
     2. If needed, it will try to scrape Sciencedirect web page.
-
     3. HEPRecord will be built.
 
+    Examples:
+        Example using ``atom_feed``::
+
+            $ scrapy crawl elsevier -a atom_feed=file://`pwd`/tests/responses/elsevier/test_feed.xml -s "JSON_OUTPUT_DIR=tmp/"
+
+        Example using ``zip`` file::
+
+            $ scrapy crawl elsevier -a zip_file=file://`pwd`/tests/responses/elsevier/nima.zip -s "JSON_OUTPUT_DIR=tmp/"
+
+        Example using ``xml`` file::
+
+            $ scrapy crawl elsevier -a xml_file=file://`pwd`/tests/responses/elsevier/sample_consyn_record.xml -s "JSON_OUTPUT_DIR=tmp/"
 
-    Example usage:
-    .. code-block:: console
 
-        scrapy crawl elsevier -a atom_feed=file://`pwd`/tests/responses/elsevier/test_feed.xml -s "JSON_OUTPUT_DIR=tmp/"
-        scrapy crawl elsevier -a zip_file=file://`pwd`/tests/responses/elsevier/nima.zip -s "JSON_OUTPUT_DIR=tmp/"
-        scrapy crawl elsevier -a xml_file=file://`pwd`/tests/responses/elsevier/sample_consyn_record.xml -s "JSON_OUTPUT_DIR=tmp/"
+    .. note::
+        * For logging, add
```

Comment:
As you are rewriting the full command right after, and not just what you have to add, the 'add' here makes no sense.